pkg/http-metrics/main.go

```diff
@@ -70,11 +103,29 @@ func main() {
 
 // ForwardRequest send all the request the the pod except for the ones having metrics/ in the path
 func ForwardRequest(res http.ResponseWriter, req *http.Request) {
+
+	klog.Infof("Received request %v", req)
+
 	requestTime := time.Now()
-	httputil.NewSingleHostReverseProxy(target).ServeHTTP(res, req)
-	responseTime := time.Now()
-	delta := responseTime.Sub(requestTime)
-	window.Append(float64(delta.Milliseconds()))
+	proxy := httputil.NewSingleHostReverseProxy(target)
```

Comment:
Why are we creating a new reverse proxy every time a request is forwarded? Should we create it at startup and then keep it for all incoming connections?